it/uk/gov/hmrc/economiccrimelevyregistration/SecondContactEmailISpec.scala

```diff
@@ -10,7 +10,7 @@ import uk.gov.hmrc.economiccrimelevyregistration.models._
 
 class SecondContactEmailISpec extends ISpecBase with AuthorisedBehaviour {
 
-  val emailMaxLength: Int = 160
+  val emailMaxLength: Int = 132
```

Comment:
Should this be using uk.gov.hmrc.economiccrimelevyregistration.forms.mappings.MaxLengths.EmailMaxLength instead?